rootfs/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
 		golang \
 		make \
 		--no-install-recommends \
-	&& export DOCKER_REGISTRY_TAG=minio \
+	&& export DOCKER_REGISTRY_TAG=master \
```

Comment:
I'm not sure why the `minio` branch is now rebased on top of master, but we should be shipping v2.4.1 with a couple patches for minio support. I would prefer to do that rather than be supporting a moving target that is likely going to break (master).

Can you please make a branch that is v2.4.1 + the commits necessary for minio and IAM support?